lib/erl_docgen/src/docgen_xml_to_chunk.erl

```diff
@@ -640,7 +640,10 @@ transform_datatype(Dom,_Acc) ->
 transform_see({See,[{marker,Marker}],Content}) ->
     AbsMarker =
         case string:lexemes(Marker,"#") of
-            [Link] -> [get(application),":",get(module),"#",Link];
+            [AppMod] when See =:= seeerl ->
+                [AppMod];
+            [Link] ->
+                [get(application),":",get(module),"#",Link];
```

Comment:
I think this breaks the links that start with an anchor, i.e. `<seeerl marker="#gc_minor_start">`.

I'm not sure why I used string:lexemes instead of string:split there, but I think the correct code would look something like this:

```
      case string:split(Marker,"#") of
            [AppMod] ->
                [AppMod];
            ["",Link] ->
                [get(application),":",get(module),"#",Link];
```

what do you think?

Reply:
good call, I'll look into it.